packages/contracts/tasks/rekey.ts

```diff
@@ -10,7 +10,7 @@ task('rekey', 'Generates a new set of keys for a test network').setAction(
       'deployer',
       'validator',
       'proxyAdminOwner',
-      'kromaBaseFeeRecipient',
+      'protocolFundRecipient',
       'kromaL1FeeRecipient',
```

Comment:
```suggestion
-kromaL1FeeRecipient
+proposerRewardVault
```

Reply:
this is not a vault, withdrawal address.